backend/lib/endpoints/posts.js

```diff
@@ -399,6 +412,8 @@ async function routes(app) {
       );
       if (updateErr) {
         throw app.httpErrors.notFound();
```

Comment:
Thanks @BMariscal for the quick turnaround. Just to note a future improvement - the user find stuff after `authenticate` & `optionalAuthenticate`  - it seems were repeating ourselves a lot I think that would be better handled in the middleware. But this is for post MVP refactoring not now.

A few suggestions:

```suggestion
        req.log.error(updateErr, "Failed unliking post");
        throw app.httpErrors.internalServerError()
```
Make this consistent with the others.